### PR TITLE
[SPRITES] Emissive mechs

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Adds emissives to mechs so they light up in the dark. All mechs that can normally be found in game have had emissive appearnces added to the

## Why It's Good For The Game
Adds a little bit of flavour to mechs, plus it looks cool.

## Images of changes

mech_emissives_open

https://github.com/user-attachments/assets/ea60db4d-cf3d-4ebf-863e-85aaa065af66

## Testing
Spawn in a Durand, glows as expected.
Enter Durand, emissive state changes to the version for an occupied mech
Exit Durand, it changes back to the open emissive state.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
imageadd: Added emissives to mechs so they now glow in the dark
/:cl:

### CODE REVIEW

**Comment by BAGELMENSK** [comment]
shmell gyeah

**Comment by Screampuff** [comment]
oh this rules

File: code/game/mecha/mecha.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	name = "Mecha"
 	desc = "Exosuit"
 	icon = 'icons/mecha/mecha.dmi'
+	var/emissive_appearance_icon = 'icons/mecha/mecha_emissive.dmi'
```
Comment (Burzah) [inline code/game/mecha/mecha.dm:7]: Could you move this down a bit with the other vars that are being declared and autodoc it? Even if it is just a small explanation (not a lot of people know about emissives).

**Review by Burzah — COMMENTED** [review]
Looks good from what I can see, just one small nit.

File: code/game/mecha/mecha.dm (hunk 1 above)
Comment (Adrer) [reply code/game/mecha/mecha.dm]: Has been addressed.

**Review by Burzah — APPROVED** [review]

**Pull request merged**